maintainers/maintainer-list.nix

```diff
@@ -2736,7 +2736,6 @@
   };
   blitz = {
     email = "js@alien8.de";
-    matrix = "@js:ukvly.org";
```

Comment:
Thanks!

Reply:
> Thanks!

You may want to drop this commit. I can fix it in another PR.

Reply:
dropped